etc/cdn-log-shipper/log-shipper.yml

```diff
@@ -151,10 +152,17 @@ Resources:
 
             // calculate listener_ids
             datas.forEach(data => {
-              const literalIp = data['x-forwarded-for'] || data['c-ip'] || '';
+              // use leftmost XFF or IP
+              const xffParts = (data['x-forwarded-for'] || '').split(',').map(s => s.trim()).filter(s => s);
+              const leftMostIp = xffParts[0] || data['c-ip'];
```

Comment:
Note that DTR does some other logic here.  It actually bothers to parse/verify the XFF pieces, and removes loopbacks/private-nets, etc.

In reality, I doubt it will make much of a difference?  This less smart code should do fine.

Reply:
I think that's fine